src/components/Auth/SignUp/SignUp.css

```diff
@@ -39,3 +39,44 @@
     color: rgb(244, 67, 54);
     margin-top: 5.7px;
 }
+
+.is-strength-0 { color: #D1462F; }
+.is-strength-1 { color: #D1462F; }
+.is-strength-2 { color: #57B8FF; }
+.is-strength-3 { color: #57B8FF; }
+.is-strength-4 { color: #2FBF71; }
+ .ReactPasswordStrength-strength-bar {
+  box-sizing: border-box;
+  height: 2px;
+  position: relative; top: 1px; right: 1px;
+  transition: width 300ms ease-out;
```

Comment:
Add display: block here, and remove from individual `.is-strength-x .ReactPasswordStrength-strength-bar`

Reply:
If `display: block` is added here, then it will be assigned to `.is-strength--1 .ReactPasswordStrength-strength-bar` also which is not required.